extensions/mssql/src/main.ts

```diff
@@ -137,35 +137,47 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
 	azdata.ui.registerModelViewProvider('bdc-endpoints', async (view) => {
 
 		const endpointsArray: Array<Utils.IEndpoint> = Object.assign([], view.serverInfo.options['clusterEndpoints']);
+		endpointsArray.forEach(endpointInfo => {
+			endpointInfo.isHyperlink = true;
+			endpointInfo.hyperlink = 'https://' + endpointInfo.ipAddress + ':' + endpointInfo.port;
+
+		});
 		if (endpointsArray.length > 0) {
-			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy');
+			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy' || e.serviceName === 'mgmtproxy');
```

Comment:
Did this change in a recent CTP?

Reply:
Yes, I sent out an email regarding this and added you to it.